project/Build.scala

```diff
@@ -238,6 +238,7 @@ object Build {
     scalaModuleInfo ~= (_.map(_.withOverrideScalaVersion(false))),
 
     libraryDependencies += "com.novocode" % "junit-interface" % "0.11" % Test,
+    libraryDependencies += "io.get-coursier" % "coursier_2.13" % "2.0.0-RC5-6" % Test,
```

Comment:
To depend on a Scala 2 library without having to write the suffix manually, you can use `withDottyCompat` (https://github.com/lampepfl/dotty-example-project#getting-your-project-to-compile-with-dotty). But I think it'd be better if we could rewrite the test to not depend on junit, otherwise we might have troubles if we upgrade to junit 5 for example. We should be able to copy-paste https://github.com/junit-team/junit4/blob/master/src/main/java/org/junit/AssumptionViolatedException.java and simplify it.